PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -114,6 +116,7 @@ struct HfCandidateSelectorD0 {
       } else {
         hfMlResponse.setModelPathsLocal(onnxFileNames);
       }
+      hfMlResponse.cacheInputFeaturesIndices(namesInputFeatures);
       hfMlResponse.init();
       outputMl.assign(((std::vector<int>)cutDirMl).size(), -1.f); // dummy value for ML output
```

Comment:
Everything is okay on my side. Just a minor comment, now you don't need this line anymore as the `outputMl` vector (defined above `init`) is going to be passed by reference in `isSelectedMl`.